bin/run-tests.sh

```diff
@@ -1,13 +1,3 @@
 #!/usr/bin/env sh
 
-if [ -z "$1" ]; then
-  test_name="ckanext.datagovuk"
-  coverage=" --with-coverage --cover-package=ckanext.datagovuk --cover-inclusive --cover-erase --cover-tests"
-else
-  test_name=$1
-  coverage=""
-fi
-
-# nosetests -v --nologcapture --with-pylons=${TEST_CKAN_INI:-test.ini} --with-randomly --ckan $coverage $test_name
-
-pytest --ckan-ini=test.ini $coverage $test_name
+pytest --ckan-ini=test.ini tests --cov=ckanext.datagovuk --cov-report=term-missing --disable-pytest-warnings -v
```

Comment:
Would be great if we could document these arguments in the commit message - some new ones here I think.

Reply:
OK, I'll add something to the commit message